lib/mongodb.js

```diff
@@ -797,6 +797,33 @@ function hasNearFilter(where) {
   return false;
 }
 
+MongoDB.prototype.buildLookup = function (relation, include, model) {
+  var self = this;
```

Comment:
Here I can see that you need 

  - relation.type
  - relation.model
  - relation.foreignKey
  - relation.through

Try provide them in `options` like:

```js
options: {
  mongodbAggregateConfig: {
    type: 'String'
    through: 'Boolean', // Seems now you don't handle the situation with a through model
    foreignKey: 'String',
    model: 'String'
  }

}
```